docs/development.md

```diff
@@ -1,418 +1,659 @@
-2020-10-15T11:32:04.890+0100    INFO    controller-runtime.controller   Starting Controller     {"controller": "securitypolicy"}
-2020-10-15T11:32:04.890+0100    INFO    controller-runtime.controller   Starting workers        {"controller": "securitypolicy", "worker count": 1}
-2020-10-15T11:32:04.890+0100    INFO    controller-runtime.controller   Starting Controller     {"controller": "apidefinition"}
-2020-10-15T11:32:04.890+0100    INFO    controller-runtime.controller   Starting workers        {"controller": "apidefinition", "worker count": 1}
-2020-10-15T11:32:04.890+0100    INFO    controller-runtime.controller   Starting Controller     {"controller": "webhook"}
-2020-10-15T11:32:04.890+0100    INFO    controller-runtime.controller   Starting workers        {"controller": "webhook", "worker count": 1}
+kubectl apply -f config/samples/httpbin.yaml 
 ```
-
-### 3. Generate & install the CRDs and run the Operator from source
-
-A) in root of directory, run this command to install the CRDs into the cluster.
-```bash
-make install
-```
-
-### 4. Add API definition through command line
-
-Add an API definition using kubectl apply
-
-```bash
-$ kubectl apply -f config/samples/httpbin.yaml
+<details><summary>SHOW EXPECTED OUTPUT</summary>
+<p>
+<pre>
 apidefinition.tyk.tyk.io/httpbin created
-```
+</pre>
+</p>
+</details>
 
-### 4. Get the new resource
+check that your api definition was applied and it works
 
-```bash
-kubectl get tykapis                                        
-NAME      PROXY.LISTENPATH   PROXY.TARGETURL      ENABLED
-httpbin   /httpbin           http://httpbin.org   true
 ```
-
-### 5. Try it out
-
-```bash
-curl localhost:8000/httpbin/get
+curl http://localhost:8000/httpbin/headers
+```
+<details><summary>SHOW EXPECTED OUTPUT</summary>
+<p>
+<pre>
 {
-  "args": {}, 
   "headers": {
     "Accept": "*/*", 
     "Accept-Encoding": "gzip", 
     "Host": "httpbin.org", 
-    "User-Agent": "curl/7.71.1", 
-    "X-Amzn-Trace-Id": "Root=1-5f64fdeb-db917d73dd04463839339047"
-  }, 
-  "origin": "127.0.0.1, 94.14.220.241", 
-  "url": "http://httpbin.org/get"
+    "Referer": "", 
+    "User-Agent": "Mozilla/5.0 (compatible; MSIE 9.0; Windows NT 6.1; Trident/5.0)", 
+    "X-Amzn-Trace-Id": "Root=1-5fc0c7f4-4fe147d429feac260d185764"
+  }
 }
-```
+</pre>
+</p>
+</details>
 
-### 5. Enable Authentication
 
-```bash
-cat ./config/samples/httpbin.yaml | yq w - spec.use_keyless false > ./config/samples/httpbin_protected.yaml
-```
+#### Community edition
 
-### 6. Try it out
+***expose our gateway locally***
 
-```bash
-kubectl apply -f ./config/samples/httpbin_protected.yaml                            
-apidefinition.tyk.tyk.io/httpbin configured
+Run this in a separate terminal
 
-curl localhost:8000/httpbin/get                   
-{
-  "error": "Authorization field missing"
-}                    
 ```
+kubectl port-forward svc/gw 8000:8000  -n ykce-control-plane
```

Comment:
namespace is wrong. needs to be `tykce-control-plane`

Reply:
done d64823d